src/main/java/core/basesyntax/service/impl/FruitTransactionParserImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.FruitTransactionParser;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitTransactionParserImpl implements FruitTransactionParser {
+    private static final int TYPE_INDEX = 0;
+    private static final int FRUIT_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private static final String COLUMN_SEPARATOR = ",";
+
+    @Override
+    public List<FruitTransaction> parseTransactions(String transactions) {
+        String[] transactionsArray = transactions.split(System.lineSeparator());
+        return Arrays.stream(transactionsArray, 1, transactionsArray.length)
```

Comment:
You should use a constant here for magic number 1 (kind of FIRST_TRANSACTION_INDEX or maybe smth more native)